ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -4,6 +4,7 @@
 //
 
 import CoreGraphics
+import UIKit
 
 /// Represents a two-part shadow as used by FluentUI.
 public struct ShadowInfo: Equatable {
```

Comment:
Optional: just an observation reading some of your code, it seems like some of it could be simplified by splitting the ShadowInfo struct into two structs, one that's just 'color, blur, x, y' and then another that's just 'shadow1, shadow2' (or in FURN we call the two shadows 'key' and 'ambient'). Then you might not need as many isShadowOne? checks.

Ex.
```
initializeShadowLayer(layer: shadow1, shadow: shadowInfo, view: view, isShadowOne: true)
initializeShadowLayer(layer: shadow2, shadow: shadowInfo, view: view)
```
could be replaced with
```
initializeShadowLayer(layer: shadow1, shadow: shadowInfo.shadow1, view: view)
initializeShadowLayer(layer: shadow2, shadow: shadowInfo.shadow2, view: view)
```

And the function itself would be a little simpler

Reply:
@laminesm were you planning to make this change?